cylc/flow/scripts/trigger.py

```diff
@@ -17,43 +17,46 @@
 
 """cylc trigger [OPTIONS] ARGS
 
-Force task(s) to run regardless of prerequisites, even in a paused workflow.
+Manually trigger a group of one or more tasks, automatically satisfying any
+off-group dependencies and respecting dependencies within the group.
 
-Triggering a task that is not yet queued will queue it.
+Group trigger is typically the easiest way to rerun a sub-graph. It erases
+flow history to allow the rerun; it identifies and triggers group start tasks
+to start the flow; and it identifies and satisfies any off-group prerequisites
+that would cause a stall.
 
-Triggering a queued task runs it immediately.
+Triggering an unqueued task queues it; triggering a queued task runs it; so you
+may need to trigger unqueued tasks twice to run them now if the queue is full.
 
-Cylc queues restrict the number of jobs that can be active (submitted or
-running) at once. They release tasks to run when their active task count
-drops below the queue limit.
+Preparing, submitted, and running tasks can't be retrigged until finished.
```

Comment:
I think this might confuse becuase tasks inside a sub-graph can be retriggered under these circumstances, and will be killed:
```suggestion
Tasks at the start of a retriggered sub-graph (including single tasks) cannot
be retriggered whilst in preparing, submitted, or running states.
```

P.S. Even if you disagree, check spelling of "retriggered".